### PR TITLE
CoVOST2 recipes

### PR DESCRIPTION
**Pull request by ftshijt**
Will upload models and results (focus on es_en for current models)

### CODE REVIEW

**Review by sw005320 — APPROVED** [review]
LGTM!
Once you finish the result part, I'll merge this PR.

File: egs2/covost2/asr1/run.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+./asr.sh \
+    --ngpu 1 \
+    --local_data_opts "--stage 0 --src_lang ${src_lang} --tgt_lang ${tgt_lang}" \
+    --use_lm false \
+    --lm_config "${lm_config}" \
+    --token_type bpe \
+    --nbpe $nbpe \
+    --feats_type raw \
+    --speed_perturb_factors ${speed_perturb_factors} \
```
Comment (chintu619) [inline egs2/covost2/asr1/run.sh:92]:
```suggestion
    --speed_perturb_factors "${speed_perturb_factors}" \
```
Without double quotes, the perturbation factors are treated as additional inputs to `asr.sh`, which throws `Error: No positional arguments are required.`

File: egs2/covost2/st1/run.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+fi
+
+if [ ${tgt_lang} == ja ] || [ ${tgt_lang} == zh-CN ]; then
+    tgt_nbpe=4000
+fi
+
+./st.sh \
+    --ngpu 1 \
+    --local_data_opts "--stage 0 --src_lang ${src_lang} --tgt_lang ${tgt_lang}" \
+    --speed_perturb_factors ${speed_perturb_factors} \
```
Comment (chintu619) [inline egs2/covost2/st1/run.sh:94]:
```suggestion
    --speed_perturb_factors "${speed_perturb_factors}" \
```

File: egs2/covost2/st1/run.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    --use_lm false \
+    --feats_type raw \
+    --audio_format "flac.ark" \
+    --use_lm false \
```
Comment (chintu619) [inline egs2/covost2/st1/run.sh:98]:
```suggestion

```
repeat of Line 95

File: egs2/covost2/mt1/run.sh [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    tgt_nbpe=4000
+fi
+
+./mt.sh \
+    --ngpu 1 \
+    --local_data_opts "--stage 0 --src_lang ${src_lang} --tgt_lang ${tgt_lang}" \
+    --use_lm false \
+    --feats_type raw \
+    --audio_format "flac.ark" \
+    --use_lm false \
```
Comment (chintu619) [inline egs2/covost2/mt1/run.sh:83]:
```suggestion

```
Repeat of Line 80

File: egs2/covost2/asr1/run.sh (hunk 1 above)
Comment (ftshijt) [reply egs2/covost2/asr1/run.sh:92]: Thanks @chintu619

**Comment by sw005320** [comment]
Thanks, @chintu619 for carefully checking the recipe.
@ftshijt!, is it ready for merge?

**Comment by ftshijt** [comment]
> Thanks, @chintu619 for carefully checking the recipe. @ftshijt!, is it ready for merge?

I'm still waiting for my asr model to finish training and then test it with an asr-pretraining result. But I think it can be merge now, I can submit the PR with results and pre-trained model later

**Comment by sw005320** [comment]
OK, I'll merge it. 
Please make a follow up PR then.

**Pull request merged**